engine/hatchery/stats.go

```diff
@@ -42,9 +42,19 @@ func (c *Common) initStats(hatcheryName string) error {
 	label = fmt.Sprintf("cds/%s/%s/disabled_workers", c.ServiceName(), hatcheryName)
 	c.stats.DisabledWorkers = stats.Int64(label, "number of disabled workers", stats.UnitDimensionless)
 
+	label = fmt.Sprintf("cds/%s/%s/starters/spawning", c.ServiceName(), hatcheryName)
+	c.stats.StartersSpawning = stats.Int64(label, "starters spawning a worker", stats.UnitDimensionless)
```

Comment:
i'm not sure i understand the metric, is it the number of goroutines spawning a worker ?

Reply:
a starter (= instance of a goroutine) have a label. example: hatchery_starter="9"

so, 

starters_spawning{hatchery="cds-hatchery-local",hatchery_name="hatcheryLocal",hatchery_starter="9"} 2

starters_waiting{hatchery="cds-hatchery-local",hatchery_name="hatcheryLocal",hatchery_starter="9"} 2

say that goroutine n°9 had spawned 2 workers and do nothing now (as waiting = spawning).

We'll discuss about that tomorrow, I'm not 100% sure that it will be really useful to understand the life of  goroutine "starter"